plugins/inventory/flightctl.py

```
@@ -268,6 +273,109 @@ def _create_tmp_crt(self, encoded_data):
+            )
+
+        if default_name:
+            for p in oidc_providers:
+                name = (p.get("metadata") or {}).get("name")
+                if name == default_name:
+                    return p
+
+        return oidc_providers[0]
+
+    def _oidc_password_grant(self, host: str, username: str, password: str,
+                             verify_ssl: bool, ca_path: str | None = None) -> str:
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)